### DISCUSSION TITLE
Field validation rule

### DISCUSSION THREAD

**Discussion by skiedude** [Budibase/budibase#14881] [Help] [answered]
I have a field where the default value is set to "0", I created a validation rule that has
![image](https://github.com/user-attachments/assets/cf59da3f-6604-427b-83cd-2a68cc1f369a)

So the value must be set to 1, however I'm able to submit the form without updating this field to 1 (so its showing in the database as the default of 0) and I'm not sure why this validation rule isn't being enforced. 

All other fields work fine.

**Comment by mikesealey** [comment, marked as answer]
How is the form submitted? 

If you're using a "Submit" button with On-CLick actions you'll also want to add a "Validate Form" action. https://docs.budibase.com/docs/form-actions#validate-form
